package.json

```diff
@@ -1,10 +1,11 @@
 {
-  "name": "js-starter",
-  "version": "0.0.1",
-  "description": "A boilerplate for starting js code",
+  "name": "agromart-app",
+  "version": "0.1",
```

Comment:
I tried starting the app with npm but because the version is `0.1` it threw an error saying `Invalid version: "0.1"` because npm uses semantic versioning. You can read up about it here=> https://semver.org/.
I suggest you change it to say `0.1.0` to accommodate those who use npm also.

Reply:
well noted

Reply:
fixed